servicechain-docker/scn/additional-node.conf

```diff
@@ -0,0 +1,21 @@
+
+# docker-compose
+NETWORK=""
+DATA_DIR="/klaytn"
+LOG_DIR="$DATA_DIR/log"
+RPC_ENABLE=1
+RPC_API="db,klay,net,web3,miner,personal,txpool,debug,admin,istanbul,subbridge"
+NETWORK_ID="9999"
+NO_DISCOVER=1
```

Comment:
Please add `VERBOSITY=5` to the all entities.

Reply:
Oh, Good idea, thank you~